.github/workflows/build-git-installers.yml

```diff
@@ -807,12 +804,12 @@ jobs:
           tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
 
-      - name: Download GPG public key signature file
-        run: |
-          az keyvault secret show --name "$GPG_PUBLIC_KEY_SECRET_NAME" \
-            --vault-name "$AZURE_VAULT" --query "value" --output tsv |
-            base64 -d >msft-git-public.asc
-          mv msft-git-public.asc deb-package
+      - name: Download Linux GPG public key signature file
+        uses: ./.github/actions/akv-secret
+        with:
+          vault: ${{ secrets.AZURE_VAULT }}
+          secrets: |
+            ${{ secrets.LINUX_GPG_PUBLIC_SECRET_NAME }} base64> deb-package/msft-git-public.asc
```

Comment:
The commit message still talks about using `az keyvault` directly; Would you mind adjusting it, please?